controllers/front/ListComments.php

```diff
@@ -71,6 +71,7 @@ public function display()
 
             if ($isLastNameAnonymous) {
                 $productComment['customer_name'] = $this->anonymizeName($productComment['customer_name']);
+                $productComment['lastname']= '...';
```

Comment:
what does `$productComment['customer_name']` contain ? is it the first name ? in this case we should contain `customer_name` and `lastname` before sending it to the `anonymizeName` method.

If it's just a name I guess it's fine :thinking:

Reply:
I'm quite sure that if we want to improve Lastname anonymizing, we need to do it inside `function anonymizeName` (lines 96 - 112), or before calling it at line 73.

Reply:
@matthieu-rolland  `customer_name` contains either `first_name + last_name` or `first_name + last_name[0]...`, but data in json response contains everything - `customer_name, first_name, last_name` with full `last_name` - thus it's not anonymized.